docs/websocket.md

```diff
@@ -586,3 +586,154 @@ eg:
+        "createdAt": "2020-03-25T21:46:31.294Z",
+        "transactionHash": "0x87bfd011dc8dcc43c512d19686d41ea9a56da3fc4cc332f245615f289d8702e0",
+        "status": "PENDING",
+        "market": "DAI-USDC",
+        "side": "SELL",
+        "price": "0.0000000000010231",
+        "amount": "2929446700000000081920",
+        "makerOrderId": "0x5060c37571e2d243961623a8bc0d76fd63c46387ccf37241c4ee95a167e02e72",
+        "makerAccountOwner": "0x97ef7039309b938e2893d97ef75c0ceebccfbd55",
+        "makerAccountNumber": "0",
+        "takerOrderId": "0x8cd16a9964d83707d840cd6e26f54ab4ca425176f479d9fd1b8642fbc8f50a2c",
+        "takerAccountOwner": "0x8ebab0129ffea1145a505d8d6d5d046770dd97e6",
+        "takerAccountNumber": "0"
+      },
+      {
+        "uuid": "39c55c38-4829-4344-a723-dc44f867019d",
+        "createdAt": "2020-03-25T21:46:31.290Z",
+        "transactionHash": "0x87bfd011dc8dcc43c512d19686d41ea9a56da3fc4cc332f245615f289d8702e0",
+        "status": "PENDING",
+        "market": "DAI-USDC",
+        "side": "SELL",
+        "price": "0.0000000000010216",
+        "amount": "1002063782644693991424",
+        "makerOrderId": "0x289b2cb5ad4673beff2b1b8819a7a17df7d1ddfecde352cd53344343787bbaaf",
+        "makerAccountOwner": "0x97ef7039309b938e2893d97ef75c0ceebccfbd55",
+        "makerAccountNumber": "0",
+        "takerOrderId": "0x8cd16a9964d83707d840cd6e26f54ab4ca425176f479d9fd1b8642fbc8f50a2c",
+        "takerAccountOwner": "0x8ebab0129ffea1145a505d8d6d5d046770dd97e6",
+        "takerAccountNumber": "0"
+      },
+    ],
+  }
+}
+```
+
+#### Updates
+
+New trades are sent in array form.
+
+A trade executed for the market:
+```json
+{
+  "type": "channel_data",
+  "connection_id": "0b88ebf2-98ec-4de6-b781-62e1648ae657",
+  "message_id": 1277,
+  "channel": "trades",
+  "id": "DAI-USDC",
+  "contents": {
+    "updates": [
+      {
+        "uuid": "7cb4e283-20bf-4ccf-a15b-dc84b86965dd",
+        "createdAt": "2020-03-25T21:46:23.351Z",
+        "transactionHash": "0xe7cd5f4fafc05c707205d8090d9cd592a87daec3977c5d1c7f0953c68f562bf6",
+        "status": "CONFIRMED",
+        "market": "DAI-USDC",
+        "side": "BUY",
+        "price": "0.0000000000010245",
+        "amount": "1024685243041289076736",
+        "makerOrderId": "0x9bd33ad14caa817135dd92d062b449ffa477f75c4f28f85825e752148e22a96e",
+        "makerAccountOwner": "0x97ef7039309b938e2893d97ef75c0ceebccfbd55",
+        "makerAccountNumber": "0",
+        "takerOrderId": "0x195a54220932ccb9416e9eba6189ffee4248b2e37b403f340982ba695c55a311",
+        "takerAccountOwner": "0x97ef7039309b938e2893d97ef75c0ceebccfbd55",
+        "takerAccountNumber": "0"
+      }
+    ]
+  }
+}
+```
+
+#### Unsubscribing
+
+```json
+{
+  "type": "unsubscribe",
+  "channel": "trades",
+  "id": "DAI-USDC"
+}
+```
+
+|Field Name|JSON type|Description|
+|----------|---------|-----------|
+|type|string|Must be set to "unsubscribe"|
+|channel|string|The channel to unsubscribe from|
```

Comment:
nit: must be trades in this case probably. Anyway if this is common to all websockets, then don't need to repeat it here, just put it in an easy-to-access place in the websockets section

Reply:
Same with `Subscribing` and `Response`. No need to have this in all sections

Reply:
I think it's easier for readers to jump to a section and immediately see the fields for whatever they're trying to do. The line `|channel|string|The channel to unsubscribe from|` is used throughout websocket.md